docs/index.md

```diff
@@ -25,7 +25,7 @@ terraform {
   required_providers {
     fastly = {
       source = "fastly/fastly"
-      version >= "0.38.0"
+      version >= "0.39.0"
```

Comment:
Hi @Integralist , I'm not sure which one you're referring to:
https://github.com/fastly/terraform-provider-fastly/tree/main/examples

Reply:
Ah nevermind. So in the RELEASE.md I reference this PR...

https://github.com/fastly/terraform-provider-fastly/pull/498/files#diff-8b82d3e2f4622aadbef4944726ecc689e77e5053cf53972037811efb4b037b0f

Which modifies the file I mentioned.

Seems like since that PR we made the file into a template...

https://github.com/fastly/terraform-provider-fastly/blob/main/examples/index-fastly-provider.tf.tmpl#L6